test/fisher.jl

```diff
@@ -107,4 +108,14 @@ show(IOBuffer(), t)
 t = HypothesisTests.FisherExactTest(1, 1, 1, 1)
 @test HypothesisTests.pvalue(t, tail=:both) <= 1
 show(IOBuffer(), t)
+
+# constructor test for matrix and vectors
+x=[1,1,1,0,0,0,0,0,1,1]
+y=[1,1,0,0,1,0,1,1,1,1]
+
+d = counts(x,y)
+
+FisherExactTest(d)
```

Comment:
Please check that results are the same for both calls, and that they are equal to what you get when passing `a`, `b`, `c` and `d` manually.